hoomd/md/pytest/test_potential.py

```diff
@@ -573,19 +651,20 @@ def test_run(simulation_factory, lattice_snapshot_factory, valid_params):
                                                      len(snap.particles.types),
                                                      snap.particles.N)
     sim = simulation_factory(snap)
-    integrator = md.Integrator(dt=0.005)
+    _check_for_skip(sim, valid_params.pair_potential)
+
+    integrator = hoomd.md.Integrator(dt=0.005)
     integrator.forces.append(pot)
     integrator.methods.append(md.methods.Langevin(hoomd.filter.All(),
                                                   kT=1, seed=1))
     sim.operations.integrator = integrator
-    sim.run(0)
-    for nsteps in [3, 5, 10]:
-        old_snap = sim.state.snapshot
-        sim.run(nsteps)
-        new_snap = sim.state.snapshot
-        if new_snap.exists:
-            assert not np.allclose(new_snap.particles.position,
-                                   old_snap.particles.position)
+    sim.operations._schedule()
+    old_snap = sim.state.snapshot
+    sim.run(2)
+    new_snap = sim.state.snapshot
+    if new_snap.exists:
+        assert not np.allclose(new_snap.particles.position,
+                               old_snap.particles.position)
 
 
 def test_energy_shifting(simulation_factory, two_particle_snapshot_factory):
```

Comment:
Why is this test being skipped now?

It mentions a bug, has the bug been fixed? If it has, we shouldn't be skipping this test anymore.

Reply:
@b-butler added this skip.